pkg/apply/ResourceInfos.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Kubernetes Authors.
```

Comment:
Files should be snake case, not camel case.

Reply:
Will do